package.json

```diff
@@ -34,7 +34,8 @@
   "license": "MIT",
   "dependencies": {
     "http-link-header": "^0.8.0",
-    "parse5": "^3.0.2"
+    "parse5": "^3.0.2",
+    "util.promisify": "^1.0.0"
```

Comment:
oops what about yarn.lock? can you generate it?

Reply:
Done